src/Synolia/Bundle/FavoriteBundle/Resources/views/layouts/default/page/layout.yml

```diff
@@ -0,0 +1,5 @@
+layout:
+    actions:
+        - '@setBlockTheme':
+            themes: '@SynoliaFavorite/layouts/default/page/layout.html.twig'
+    conditions: 'context["isFavoriteEnabled"] == true'
```

Comment:
the conditions is not needed here as it's the bloc types content

Reply:
Condition deleted